src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { VerifiedIcon } from 'nft/components/icons'
+import { CollectionInfoForAsset, GenieAsset } from 'nft/types'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+
+const MAX_WIDTH = 560
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: ${({ theme }) => `calc(100vh - ${theme.navHeight}px)`};
+  align-items: center;
+  padding: 22px 20px 0px;
+  gap: 26px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    gap: 64px;
+    padding-top: 28px;
+  }
+
+  @media screen and (min-width: ${BREAKPOINTS.xl}px) {
+    flex-direction: row;
+    padding-top: 0px;
+    padding-bottom: ${({ theme }) => `${theme.navHeight}px`};
+    gap: 80px;
+  }
+`
+
+const InfoContainer = styled(Column)`
```

Comment:
nit: `ColumnCenter` exported from the same file as Column has width 100% and align center

Reply:
Can this and `InfoDetailsContainer` be combined? `InfoContainer` only has 1 child element which is `InfoDetailsContainer` so it's column attributes aren't used

Reply:
> Can this and InfoDetailsContainer be combined?

Or will InfoContainer be expanded upon later?